packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -86,17 +83,34 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 					__( 'Free' )
 				) : (
 					<>
-						<span className="domain-picker__free-text"> { __( 'Included in plans' ) } </span>
-						<span className="domain-picker__price-is-paid">
+						<span className="domain-picker__price-long">
+							{
+								/* translators: %s is the price with currency. Eg: $15/year. */
+								createInterpolateElement(
+									sprintf(
+										__( 'Included in paid plan, <strikethrough>%s/year</strikethrough>' ),
```

Comment:
I think the `translators:` comment needs to go right above this line — it could work like this too, not actually sure :-)

Reply:
Good point!